README.md

```diff
@@ -113,7 +113,7 @@ RE(queue_server_plan())
 On your host machine run:
```

Comment:
I think just noting that this relies on the httpie command line client would be great, i.e. `(use your package manager to install httpie)`